src/OpenTelemetry.Exporter.Jaeger/README.md

```diff
@@ -45,6 +44,38 @@ See the
 [`TestJaegerExporter.cs`](../../examples/Console/TestJaegerExporter.cs)
 for an example of how to use the exporter.
 
+## Resources and Process Tags
+
+The attributes of a [Resource](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/resource/sdk.md)
+ are represented in Jaeger as `ProcessTags` that associate with an
+exported span. The exception is the [required](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/resource/semantic_conventions/README.md#semantic-attributes-with-sdk-provided-default-value)
+`service.name` attribute which accompanies each exported span as shown below.
```

Comment:
This text is a bit misleading, IMO. It sort of sounds like when we export spans to Jaeger this data is copied on every span, which would be expensive and actually why it was removed from Zipkin. What actually happens is the "process" information is sent once per batch with the spans for that batch. It is only in the UI where it is displayed on the spans. It might be worth clarifying that point just so we don't discourage anyone from using Resources with Jaeger? 🤷

Reply:
Great point, I'll clarify this to be more explicit.